### PR TITLE
Update TestCoin.move

### PR DESCRIPTION
**Pull request by gregnazario**
No reason for 'transfer' to own the 'from' signer. Should be &signer.

This is a rebased version of https://github.com/aptos-labs/aptos-core/pull/327 by @magnum6actual

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/TestCoin.move [hunk 1]
```diff
@@ -162,20 +162,24 @@ module AptosFramework::TestCoin {
         borrow_global<Balance>(owner).coin.value
     }
 
+	public(script) fun transfer(from: signer, to: address, amount: u64) acquires Balance, TransferEvents {
+		transfer_internal(&from, to, amount);
+	}
+
```
Comment (davidiw) [inline aptos-move/framework/aptos-framework/sources/TestCoin.move:168]: spacing looks off here, make sure spaces not tabs

**Review by davidiw — APPROVED** [review]
check spacing before landing

File: aptos-move/framework/aptos-framework/sources/TestCoin.move (hunk 1 above)
Comment (gregnazario) [reply aptos-move/framework/aptos-framework/sources/TestCoin.move:168]: Oh, let me run the formatter

**Pull request merged**